lxd/storage/drivers/utils.go

```diff
@@ -739,7 +739,7 @@ func BTRFSSubVolumesGet(path string) ([]string, error) {
 
 // btrfsIsSubvolume checks if a given path is a subvolume.
 //
-// Deprecated: Use IsSubvolume from the Btrfs driver instead.
+// Deprecated: Use btrfs.isSubvolume().
 func btrfsIsSubVolume(subvolPath string) bool {
```

Comment:
can we remove this and update where its used to use btrfs.isSubVolume? Are the implementations different?

Reply:
I tried something, please have a look as I'm unsure that's the right way to go.